addons/mail/models/mail_channel.py

```diff
@@ -168,6 +176,14 @@ def _compute_member_count(self):
         for channel in self:
             channel.member_count = member_count_by_channel_id.get(channel.id, 0)
 
+    @api.depends('channel_type')
+    def _compute_group_public_id(self):
+        group_user = self.env.ref('base.group_user')
+        groups_chats = self.filtered(lambda channel: channel.channel_type != 'channel')
```

Comment:
`chats` is incorrect, as "not channel" contains more things than just chat.

I would invert the flow, to have a variable named `channels`, and do the difference the other way around.